### PR TITLE
fixes emp grenades not deleting themselves

### LINKED ISSUE #9727: Classic EMP grenades don't get deleted after they detonate
You end up with blinking light toy that can be picked up but not used again

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #9727

### CODE REVIEW

**Comment by gbasood** [comment]
@Intigracy Do you know why empulse didn't return a positive value?

File: code/game/objects/items/weapons/grenades/emgrenade.dm [hunk 1]
```diff
@@ -4,9 +4,10 @@
...
-		..()
-		if(empulse(src, 4, 10))
-			del(src)
-		return
+/obj/item/weapon/grenade/empgrenade/prime()
+	..()
+	empulse(src, 4, 10)
+	spawn(5)
+		qdel(src)
+	return
```
Comment (PJB3005) [inline code/game/objects/items/weapons/grenades/emgrenade.dm]: Muh trailing returns.

**Comment by gbasood** [comment]
Fix the trailing return and this PR is good to go

**Comment by Intigracy** [comment]
k

**Pull request merged**